opr-core/src/util/randomuuid.ts

```diff
@@ -0,0 +1,26 @@
+import {RandomSeed, create} from 'random-seed';
+
+function d(seed: RandomSeed): string {
+  return seed.range(16).toString(16);
+}
+
+const GROUPINGS = [8, 4, 4, 4, 12];
+
+export default function getUuid(seed?: RandomSeed) {
```

Comment:
Feels weird using this in favor of the standard uuid library. Is this just for supporting the seed for testing? Are we giving up things by not supporting the uuid type?

Reply:
It's just supporting generating an id based on a seeded random number generator for development. Real code can use whatever uuid library they want.